accounts/tests.py

```diff
@@ -1,3 +1,28 @@
-# from django.test import TestCase
+import pytest
+from django.contrib.auth.models import User
+from .models import UserAccount
 
-# Create your tests here.
+
+@pytest.fixture
```

Comment:
@ifireball is it okay to create fixtures here, or should be under conftest.py?

Reply:
`conftest.py` is for sharing fixtures between multiple modules (files), if the fixtures aren't shared they should not be there.

Reply:
@mayasa3 is it shared fixture?

Reply:
It isn't shared, I just added it for my tests for the user model. I don't know if it's relevant for the others, but they haven't used it.